jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/HoverTests.java

```diff
@@ -17,70 +17,103 @@
-                    .cssClasses(Matchers.hasItem(not("show-btns"))));
+            card.getPlayerButtons().stream()
+                    .map(Button::has)
+                    .forEach(textAssert -> textAssert.cssClasses(Matchers.hasItem(not("show-btns"))));
 
             card.getHover().show();
-            card.getPlayerButtons().stream().map(Button::has).forEach(textAssert -> textAssert.cssClass("show-btns"));
+            card.getPlayerButtons().stream()
+                    .map(Button::has)
+                    .forEach(textAssert -> textAssert.cssClass("show-btns"));
+
+            card.getPlayerButtons().forEach(button -> button.core().isClickable());
             card.getPlayerButtons().forEach(HasClick::click);
 
             card.getHover().suppress();
-            card.getPlayerButtons().stream().map(Button::has).forEach(textAssert -> textAssert
-                    .cssClasses(Matchers.hasItem(not("show-btns"))));
+            card.getPlayerButtons().stream()
+                    .map(Button::has)
+                    .forEach(textAssert -> textAssert.cssClasses(Matchers.hasItem(not("show-btns"))));
         });
     }
 
-    @Test
-    public void cookingCardTest() {
-        cookingHeader.has().text(containsString("cooking utensils"));
-        cartButton.click();
+    @Test(description = "Assert that Music Card list has proper texts")
+    public void musicCardListTextsTest() {
+        SUBHEADINGS.forEach(subheading -> jdiAssert(musicHoverCards.stream()
+                .map(MusicHoverCard::getSubheading)
+                .map(IsText::getText)
+                .collect(Collectors.toList()), Matchers.hasItem(subheading)));
+    }
+
+    @Test(description = "Assert that Cooking Card price text appears only when card is being hovered")
+    public void cookingCardPriceTextAppearsWhenHoveredTest() {
+        cookingHoverCard.getPriceHover().is().notVisible();
+        cookingHoverCard.getPriceHover().show();
+        cookingHoverCard.getPriceHover().has().text(containsString(COOKING_CARD_PRICE));
+        cookingHoverCard.getPriceHover().suppress();
+        cookingHoverCard.getPriceHover().is().notVisible();
+    }
+
+    @Test(description = "Assert that Cooking Card header contains proper text")
+    public void cookingCardHeaderTextTest() {
+        cookingHoverCard.getCookingHeader().has().text(containsString(COOKING_CARD_HEADER_TEXT));
+    }
 
-        priceHover.show();
-        priceHover.has().text(containsString("$14.99"));
-        priceHover.suppress();
+    @Test(description = "Assert that Cooking Card button is clickable, and price text appears when clicking")
+    public void cookingCardCardButtonTest() {
+        cookingHoverCard.getCartButton().core().isClickable();
+        cookingHoverCard.getCartButton().click();
+        cookingHoverCard.getPriceHover().has().text(containsString(COOKING_CARD_PRICE));
+        cookingHoverCard.getPriceHover().suppress();
     }
-}
+}
```

Comment:
Please, investigate opportunity to check 'delays' on 'open/close-delay'. It's important feature of 'hover'. Thanks ^^